File: src/main/java/br/com/estimular/atendimento/repository/ConvenioPacienteRepository.java
package br.com.estimular.atendimento.repository;

import br.com.estimular.atendimento.model.ConvenioPaciente;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ConvenioPacienteRepository extends JpaRepository<ConvenioPaciente, Long> {
}

File: src/main/kotlin/com/clinica/atendimento/repository/ConvenioPacienteRepository.kt
package com.clinica.atendimento.repository

import com.clinica.atendimento.model.ConvenioPaciente
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ConvenioPacienteRepository : JpaRepository<ConvenioPaciente, Long>